libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.kt

```diff
@@ -145,8 +145,17 @@ open class LoginActivity : AppCompatActivity(), OAuthWebviewHelperEvents {
 
         val salesforceSDKManager = SalesforceSDKManager.getInstance()
 
-        // Determine if the activity was created from a deep link intent with QR code log in via UI bridge API parameters.
-        val isDeepLinkedQrCodeLogin = isDeepLinkedQrCodeLogin(intent)
+        /*
+         * For Salesforce Identity API UI Bridge support, the overriding front door bridge URL to
+         * use in place of the default initial login URL plus the optional web server flow code
+         * verifier accompanying the front door bridge URL
+         */
+        val frontDoorBridgeUrl = salesforceSDKManager.frontDoorBridgeUrl
```

Comment:
Here's the use of the new SDK manager properties.  I also remove the QR code semantic here since maybe this code could be receiving a front door URL from another means.  Note how both properties are unset after use.